src/utils/useEnumOptions.ts

```diff
@@ -0,0 +1,33 @@
+import { useAttributeDefinition } from 'scrivito'
+
+// TODO: Remove/trim, once #11238 is resolved
+export function useEnumOptions(): Array<{ title: string; value: string }> {
```

Comment:
* [ ] FYI maybe put React hooks in a hooks directory

Reply:
Hopefully this is only temporary, until #11238 lands.